### PR TITLE
Made H1 heading more descriptive

### PR DESCRIPTION
**Pull request by markeno**

### CODE REVIEW

**Comment by markeno** [comment]
@znation ... I added text to heading "Introduction" to say "Introduction to Text-Analysis"... to make the page more apparent "where" you are at... currently it appears to be the beginning of the entire user guide.

File: text/intro.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
-#Introduction
+#Introduction to Text-Analysis
```
Comment (znation) [inline text/intro.md]: Why is this `Introduction to Text-Analysis`? The left nav says `Text analysis` (no dash, no "Introduction to", no capital letter on analysis). I think they should match.

Comment (markeno) [inline text/intro.md]: Roger that… changed :-)
 <http://www.dato.com/>	 	Mark Enomoto
Designer 
mark@dato.com <mailto:mark@dato.com>	 	 <https://twitter.com/datoinc>	 <http://www.linkedin.com/company/graphlab>
 <https://www.facebook.com/datoinc>	 <http://www.youtube.com/channel/UC9T07WMdwB6IPf_dKXhVsCQ>

**Comment by znation** [comment]
Looks good to me. Merging.

**Pull request merged**